### ISSUE TITLE
pip installation is missing license

### ISSUE THREAD

**Issue by JuliaSprenger** [G-Node/odml-ui#86]
The license file is not copied during the installation, but is opened in Editor.py

```
File "python3.6/site-packages/odmlui/Editor.py", line 89, in <module>
    with open("LICENSE") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'LICENSE'
```

(Python 3, odml 1.3.2, odml-ui 1.3fix branch)

**Comment by mpsonntag** [comment]
Will be fixed as soon as #83 is merged.

**Comment by JuliaSprenger** [comment]
Are you sure? Currently I am testing already this branch (https://github.com/mpsonntag/odml-ui/tree/1.3fixes)

**Comment by mpsonntag** [comment]
Yes, the #83 PR contains the MANIFEST.in file that includes the LICENSE file when packing. I successfully tested it with vanilla python 2 / 3 and clean Anaconda environments for python 2 and 3.

**Comment by mpsonntag** [comment]
Hm, but something is definitely weird when I'm not installing via setup.py but via a local pip install... I'll check it out!

**Comment by mpsonntag** [comment]
Fixed with PR #106.
